viewer/packages/360-images/src/icons/IconCollection.ts

```diff
@@ -62,20 +70,26 @@ export class IconCollection {
   constructor(
     points: Vector3[],
     sceneHandler: SceneHandler,
-    onBeforeSceneRendered: EventTrigger<BeforeSceneRenderedDelegate>
+    onBeforeSceneRendered: EventTrigger<BeforeSceneRenderedDelegate>,
+    iconOptions?: IconsOptions
   ) {
-    const sharedTexture = this.createOuterRingsTexture();
-    const iconSpriteRadius = this._iconRadius;
-    const iconsSprites = new InstancedIconSprite(
-      points.length,
-      sharedTexture,
-      this.MIN_PIXEL_SIZE,
-      this.MAX_PIXEL_SIZE,
-      iconSpriteRadius
+    this.MAX_PIXEL_SIZE = Math.min(
+      this.DEFAULT_MAX_PIXEL_SIZE,
+      iconOptions?.platformMaxPointsSize ?? this.DEFAULT_MAX_PIXEL_SIZE
     );
+
+    const sharedTexture = this.createOuterRingsTexture();
+
+    const iconsSprites = new Image360PointsObject(points.length * 2, {
```

Comment:
Is this intended to be `points.length * 2`? Seems like it should be `points.length` to me

Reply:
It was the same in 360 image icons but will change to length. Can't see why it could require double the length of number of 360 images fetched.